PyMca5/PyMcaGui/io/TiledDataChannelTable.py

```diff
@@ -0,0 +1,191 @@
+from PyQt5 import QtWidgets
+
+from PyMca5.PyMcaGui import PyMcaQt as qt
+from PyMca5.PyMcaGui.io.SpecFileCntTable import CheckBoxItem
+
+class TiledDataChannelTable(qt.QTableWidget):
+    """
+    Creates the data channel table (second table) inside the QTiledWidget.
+    The selections for the x and y set the axes in the scan window.
+    """
+    sigTiledDataChannelTableSignal = qt.pyqtSignal(object)
+    def __init__(self, parent=None):
+        qt.QTableWidget.__init__(self, parent)
+        self.dataChannelList = []
+        self.xSelection = []
+        self.ySelection = []
+        self.monSelection = []
+        
+    def format_table(self):
+        """Sets the column headers and the size of the columns for the table."""
+        
+        # Column Labels
+        labels = ['Data Channel', 'x', 'y', 'Mon']
+        self.setColumnCount(len(labels))
+        for i in range(len(labels)):
+            item = self.horizontalHeaderItem(i)
+            if item is None:
+                item = qt.QTableWidgetItem(labels[i])
+            item.setText(labels[i])
+            self.setHorizontalHeaderItem(i, item)
+
+    def clear_table(self):
+        """Clears the table if a different scan is selected."""
+        # Clear contents of the table
+        self.setRowCount(0)
+        self.setColumnCount(0)
+
+        # Reset internal state
+        self.dataChannelList = []
+        self.xSelection = []
+        self.ySelection = []
+        self.monSelection = []
+
+        self.format_table()
+
+    def build_table(self, channelList):
+        """
+        Builds the table in QTiledWidget based on the datachannels in selected
+        scan file.
+        """
+        self.dataChannelList = channelList
+        n = len(channelList)
+        self.setRowCount(n)
+        if n > 0:
+            for i in range(n):
+                self._addLine(i, channelList[i])
```

Comment:
Pythonic loop permits a more descriptive name for the list item.

```suggestion
            for (i, channelLabel) in enumerate(channelList):
                self._addLine(i, channelLabel)
```